docs/classifier_interfaces.rst

```diff
@@ -0,0 +1,29 @@
+Classifier Interfaces
+---------------------
+
+Here we list and briefly describe the high level algorithm interfaces which SMQTK provides.
```

Comment:
Specialize the labeling of this to this module (e.g. `SMQTK` --> `SMQTK-Classifier`)